api/Requests/Corporates/CreateCorporateUserFullRequest.ts

```diff
@@ -1,6 +1,11 @@
 import { CorporateUser } from '~/api/Models/Corporates/CorporateUser'
+import { CorporateUserRole } from '~/api/Enums/Corporates/CorporateUserRole'
 
 export interface CreateCorporateUserFullRequest {
-  request: Nullable<Partial<CorporateUser>>
+  request: corporateUserRequest
   corporateId: string
 }
+
+export type corporateUserRequest = Nullable<
```

Comment:
PascalCase для типов
```suggestion
export type СorporateUserRequest = Nullable<
```